README.md

```diff
@@ -1,48 +1,521 @@
+After debugging detach from shell by pressing `CTRL + P` and `CTRL + Q`.
+
+`Q:` How to debug frontend with Chrome?  
+`A:` Place `debugger;` in code, save the file, run `make logs`, 
+wait a second or two until frontend rebuilds itself
+(you can check it with `make logs`), refresh page in browser and
+interact with it to hit that breakpoint.
+
+<br>
+
+If you have any more questions not described here then please ask us
+on `#pah` on [Slack](https://codeforpoznan.slack.com).
+
 
-username                   | password  | Django Admin access
--------------------------- | --------- | -------------------
-hello@codeforpoznan.pl     | pass123   | yes
-ola@pah.org.pl             | pass123   | no
-                           |           |
-driver@codeforpoznan.pl    | pass123   | no
-passenger@codeforpoznan.pl | pass123   | no
+<br>
 
+#### Initial admin credentials
+We have 2 default users who are always present - `hello` and `ola`,
+but you can create a few more default by running `make populate-database`.
+This command will create `driver` and `passenger` and also few other random
+users and basic entities (Cars, Projects).
+Every user that's randomly created has the same password -`pass123`.
 
+Here's a table with basic users + the default users created 
+when DB is populatd:
 
-## API documentation
-Available at `/api/docs/` URL.
-Documentation is available only to logged-in users (DRF quirk).
+username                        | password  | Django Admin access
+------------------------------- | --------- | -------------------
+hello@codeforpoznan.pl          | pass123   | yes
+ola@pah.org.pl                  | pass123   | no
+...after `populate-database`... | ...       | ...
+driver@codeforpoznan.pl         | pass123   | no
+passenger@codeforpoznan.pl      | pass123   | no
 
-## Install python modules locally
-```bash
+
+<br>
+
+#### API Documentation
+
+REST API is built with Django Rest Framework. That package has also
+autogenerated interactive documentation for its endpoints.
+You can take a look by going to
+[localhost:8080/api/docs/](http://localhost:8080/api/docs/) or
+[dev.pahfm.codeforpoznan.pl/api/docs/](https://dev.pahfm.codeforpoznan.pl/api/docs/).
+
+> NOTE: documentation is available only to logged-in users because of
+> a DjangoRestFramework quirk, so be sure to log in to Admin panel.
+
+
+<br>
+
+#### Support for Auto-Completion via Python Environments
+
+If your editor does not support hooking up to Docker containers for 
+python environments then you'll probably have to create that environment
+yourself in order to have a nice code auto completion features.
+
+```shell script
+cd pah-fm
 python3 -m venv ./backend/.venv
 source ./backend/.venv/bin/activate
-pip3 install -r requirements/dev.txt
+pip3 install -r requirements/dev.txt -r requirements/base.txt
 ```
 
 Now pip modules are located in `backend/.venv/lib/python3.7/site-packages`. 
```

Comment:
It's going to use the system default python version to create venv.

For example, in my case, it is `3.8`

Current project version:
https://github.com/CodeForPoznan/pah-fm/blob/4fe011b454c3ae75ce87f697d7254eadc58869fa/.travis.yml#L5
https://github.com/CodeForPoznan/pah-fm/blob/4fe011b454c3ae75ce87f697d7254eadc58869fa/ansible/pahfm.yml#L65

And:
https://github.com/CodeForPoznan/pah-fm/pull/500

It is not consistent, and I feel like it is not even necessary to mention that part?
Or just in a more generic way.